home/src/main/resources/rdf/abox/filegraph/dynamic-api-individuals.n3

```diff
@@ -20,66 +20,129 @@
 <https://vivoweb.org/ontology/vitro-dynamic-api/parameter/validator/IsInteger>
         a                       dynapi:validator ,
                                 dynapi_java:validators.Validator ,
-                                dynapi_java:validators.IsInteger;
+                                dynapi_java:validators.IsInteger ;
         rdfs:label              "Is integer"@en-US ;
         vitro:mostSpecificType  dynapi:validator .
 
+<https://vivoweb.org/ontology/vitro-dynamic-api/parameter/validator/IsMaxLength>
+        a                       dynapi:validator ,
+                                dynapi_java:validators.Validator ,
+                                dynapi_java:validators.IsMaxLength ;
+        rdfs:label              "Is max length"@en-US ;
+        vitro:mostSpecificType  dynapi:validator .
+
+<https://vivoweb.org/ontology/vitro-dynamic-api/parameter/validator/IsMaxValue>
+        a                       dynapi:validator ,
+                                dynapi_java:validators.Validator ,
+                                dynapi_java:validators.IsMaxValue ;
+        rdfs:label              "Is max value"@en-US ;
+        vitro:mostSpecificType  dynapi:validator .
+
+<https://vivoweb.org/ontology/vitro-dynamic-api/parameter/validator/IsMinValue>
+        a                       dynapi:validator ,
+                                dynapi_java:validators.Validator ,
+                                dynapi_java:validators.IsMinValue ;
+        rdfs:label              "Is min value"@en-US ;
+        vitro:mostSpecificType  dynapi:validator .
+
+<https://vivoweb.org/ontology/vitro-dynamic-api/parameter/validator/IsRegex>
+        a                       dynapi:validator ,
+                                dynapi_java:validators.Validator ,
+                                dynapi_java:validators.IsRegex ;
+        rdfs:label              "Is regular expression"@en-US ;
+        vitro:mostSpecificType  dynapi:validator .
+
```

Comment:
Those validators request parameters (min value, max value, regExpression, etc.), therefore I am not sure it might be defined here in individuals file. We can for instance define validators for some common values, it might be a regular expression validator for email address. I have created a new PR for validators (see here - https://github.com/vivo-project/Vitro/pull/264). Therefore, I suggest that complete this PR for ParameterTypes, and to remove this part completely for now.